### DISCUSSION TITLE
[Script request] ClickHouse

### DISCUSSION THREAD

**Discussion by BasixKOR** [community-scripts/ProxmoxVE#1127] [Request script]
### Application Name

ClickHouse

### Website

https://clickhouse.com/

### Description

ClickHouse is an open-source, real-time analytics database management system (DBMS) designed for high-performance data processing and analysis. It is proven to be useful for [log storage solution](https://clickhouse.com/use-cases/logging-and-metrics), and I think it can be a fun challenge to try on a homelab.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Qwekkeboom** [comment]
I would like to have this. This would also be great in combination with https://clickhouse.com/clickstack.
